### ISSUE TITLE
Requesting new release

### ISSUE THREAD

**Issue by Whissi** [xiph/flac#19]
Hi,

any chance to get a new release to to push out https://github.com/xiph/flac/commit/43ba7ad05f1656e885ce2f34a9a72494f45705ae and https://github.com/xiph/flac/commit/c06a44969c1145242a22f75fc8fb2e8b54c55303?

In addition please see https://bugs.gentoo.org/show_bug.cgi?id=540072#c1 which is suggesting to check for malloc failures as well.

**Comment by erikd** [comment]
Happy to do a new release, but first want to know if there are any known problems with the code that's currently in git. It would help if you could provide the files that cause problems and the command line used to trigger the problems.

**Comment by lvqcl** [comment]
I thought that cpu.c isn't fully fixed yet? (per <http://lists.xiph.org/pipermail/flac-dev/2016-June/005971.html> )

**Comment by erikd** [comment]
@lvqcl That is a separate issue but if https://github.com/xiph/flac/blob/master/src/libFLAC/cpu.c#L72 is incorrect, please raise a separate issue, or send an email to the dev list.

**Comment by hannob** [comment]
@erikd the crash sample is attached to this old sourceforge-bug:
https://sourceforge.net/p/flac/bugs/421/

Just "flac -d crash.flac" reproduces a segfault for me.

**Comment by erikd** [comment]
@hannob Git head does not crash with that file and command line.

**Comment by hannob** [comment]
@erikd no, sorry, was a misunderstanding. This is fixed in git, just not in the latest release. whissi has asked me via mail for a reproducer for that bug.

**Comment by erikd** [comment]
The new release is in the pipeline. I've announced two pre-releases on the flac-dev list which have turned up a few minor issues. There will likely be one more pre-release (in the next day or two) and then the final.
